packages/koenig-lexical/src/components/ui/SettingsPanel.jsx

```diff
@@ -50,10 +51,10 @@ export function ToggleSetting({label, description, isChecked, onChange, dataTest
 export function InputSetting({label, description, onChange, value, placeholder, dataTestId}) {
     return (
         <div className="mt-2 flex w-full flex-col justify-between gap-2 text-[1.3rem] first:mt-0">
-            <div className="font-bold text-grey-900 dark:text-grey-200">{label}</div>
+            <div className="text-grey-900 dark:text-grey-200 font-bold">{label}</div>
```

Comment:
This CSS class names order changes will result in linting errors. I think this is coming from opening the monorepo in VScode, as opposed to only opening the package (koenig-lexical). It's a bit annoying – but that's how it is atm :D Potentially some of these linting errors can be fixed by running `yarn lint --fix`

Reply:
Ohh I was wondering that (and thinking that the changed order is super weird). I'll try opening the package and running that!